src/middleware/validator/validator.ts

```diff
@@ -1,9 +1,10 @@
 import { JSONPath } from '../../utils/json'
+import type { JSONObject, JSONPrimative, JSONArray } from '../../utils/json'
 import { rule } from './rule'
 import { sanitizer } from './sanitizer'
 
 type Target = 'query' | 'header' | 'body' | 'json'
-type Type = string | number | boolean | object | undefined
+type Type = JSONPrimative | JSONObject | JSONArray | File
```

Comment:
I think this is a typo.

```suggestion
type Type = JSONPrimitive | JSONObject | JSONArray | File
```

Reply:
You're right! I fixed it